packages/desktop-client/src/components/reports/ReportRouter.js

```diff
@@ -14,6 +15,7 @@ export function ReportRouter() {
       <Route path="/net-worth" element={<NetWorth />} />
       <Route path="/cash-flow" element={<CashFlow />} />
       <Route path="/category-spending" element={<CategorySpending />} />
+      <Route path="/custom-report" element={<CustomReport />} />
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏oh sorry, I meant just the component name. We can keep the old route path. It was much cleaner before.

```
- /reports/custom-report
+ /reports/custom
```